src/main/kotlin/com/cognifide/gradle/aem/common/instance/LocalInstance.kt

```diff
@@ -194,14 +193,20 @@ class LocalInstance private constructor(aem: AemExtension) : Instance(aem) {
         FileOperations.amendFile(binScript("start", OperatingSystem.forName("windows")).bin) { origin ->
             var result = origin
 
-            // Force CMD to be launched in closable window mode.
+            // Update 'timeout' to 'ping' as of it does not work when called from process without GUI
+            result = result.replace(
+                    "timeout /T 1 /NOBREAK >nul",
+                    "ping 127.0.0.1 -n 3 > nul"
+            )
+
+            // Force AEM to be launched in background
             result = result.replace(
                     "start \"CQ\" cmd.exe /K",
-                    "start /min \"CQ\" cmd.exe /C"
+                    "cbp.exe"
```

Comment:
this is recompiled chp.exe ; to avoid false-positive anti-virus vulnerabilities reported